pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -799,14 +802,6 @@ func TestPodFailedSchedulingMultipleTimesDoesNotBlockNewerPod(t *testing.T) {
 			NominatedNodeName: "node1",
 		},
 	}
-	// Update pod condition to unschedulable.
-	podutil.UpdatePodCondition(&unschedulablePod.Status, &v1.PodCondition{
-		Type:          v1.PodScheduled,
-		Status:        v1.ConditionFalse,
-		Reason:        v1.PodReasonUnschedulable,
-		Message:       "fake scheduling failure",
-		LastProbeTime: metav1.Now(),
```

Comment:
You can remove LastProbeTime here, but please keep the rest of the code. Otherwise, the scheduling queue will not consider the pod as "unschedulable".

Reply:
Done thank you